pgcli/main.py

```diff
@@ -491,6 +494,8 @@ def run_cli(self):
                 else:
                     query = self.execute_command(document.text, query)
 
+                self.now = datetime.datetime.today()
```

Comment:
I tried it at first, but the problem is the program calls get_prompt for every single action that is done (typing one letter, erasing one letter, etc), so it keeps updating the timer every single time get_prompt is called.